sunpy/map/mapbase.py

```diff
@@ -518,6 +518,28 @@ def _new_instance(cls, data, meta, plot_settings=None, **kwargs):
             new_map.plot_settings.update(plot_settings)
         return new_map
 
+    @classmethod
+    def fix_duplicate_notes( cls,subclass_doc, cls_doc):
```

Comment:
Actually I was wrong, can this be moved out into a sunpy.util location. 

Then we can import in the class instead. Map's code base is pretty long and I don't want to make it longer with functions that are not needed in this file.

Reply:
understandable, did the changes!